packages/mobile/src/services/session.js

```diff
@@ -42,11 +43,18 @@ const create = async ({
     redirectUri,
     data,
   });
+
   axios.defaults.headers.common = {
     Authorization: `Bearer ${response.data.token}`,
   };
   await setSession(response.data);
-  await setHasAlreadyLogged(true);
+
+  if (response.data.email === env.DEFAULT_USER_EMAIL) {
```

Comment:
We can simplify and remove if/else: 
 await setHasAlreadyLogged(response.data.email === env.DEFAULT_USER_EMAIL)